src/JKetelaar/Kiyoh/Models/Review.php

```diff
@@ -129,6 +136,22 @@ public function setRating(float $rating): void
         $this->rating = $rating;
     }
 
+    /**
+     * @return string
```

Comment:
Could you add a line of extra information indicating what `comment` means and returns?

Reply:
Yes!